WSL/user-support.md

```diff
@@ -1,137 +1,53 @@
+Once you have enabled WSL and installed a Linux distribution from the Microsoft Store, the first step you will be asked to complete when opening your newly installed Linux distribution is to create an account, including a **User Name** and **Password**.
 
-1. It is your default user -- it signs-in automatically on launch.
-1. It is Linux administrator (a member of the sudo group) by default.
+- This **User Name** and **Password** is specific to your Linux distribution and has no bearing on your Windows user name.
 
-Each Linux distribution running on the Windows Subsystem for Linux has its own Linux user accounts and passwords.  You will have to configure a Linux user account any time you add a distribution, reinstall, or reset.  Linux user accounts are not only independent per distribution, they are also independent from your Windows user account.
+- Once you create this **User Name** and **Password**, the account will be your default user for the distribution and automatically sign-in on launch.
 
-## Resetting your Linux password
+- This account will be considered the Linux administrator, with the ability to run `sudu` (Super User Do) administrative commands.
 
-If you have access to your Linux user account and know your current password, change it using Linux password reset tools of that distribution -- most likely `passwd`.
+- Each Linux distribution running on the Windows Subsystem for Linux has its own Linux user accounts and passwords.  You will have to configure a Linux user account every time you add a distribution, reinstall, or reset.
 
-If that's not an option, depending on the distribution, you may be able to reset your password by resetting the default user.
+## Reset your Linux password
 
-WSL offers a default user tag to identify which user account automatically logs in when you start a WSL.  Since many distributions include commands to set the default user to root and also a root user with no password set, changing the default user to root is a handy tool for things like password reset.
+To change your password, open your Linux distribution (Ubuntu for example) and enter the command: `passwd`
 
-### For Creators Update and earlier
-If you're running Windows 10 Creators update or earlier, you can change the default Bash user by running the following commands:
+You will be asked to enter your current password, then asked to enter your new password, and then to confirm your new password.
 
-1. Change the default user to `root`:
+### Forgot your password
 
-    ```console
-    C:\> lxrun /setdefaultuser root
-    ```
+If you forgot the password for your Linux distribution, try this: 
 
-1. Run `bash.exe` to now login as `root`:
+For Ubuntu:
+1. Open PowerShell and use it to change the default user name of your Ubuntu distribution to root with: `ubuntu config --default-user root`
```

Comment:
You don't need to do this change default user business. You can actually run as root directly!
`wsl -u root` will start you off in your default distro as root, and `wsl -d Debian -u root` will run you as root in the Debian distro.

From there you can go in and run the `passwd` command and you won't need to reset your default user after that. :) 

This should simplify this whole section!